### PR TITLE
path

### PR DESCRIPTION
**Pull request by Tigee03**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -33,69 +33,56 @@ hr{
...
 }
 .drag-area img{
-    height: 100%;
-    width: 100%;
-    object-fit: cover;
-    border-radius: 5px;
-}
-
-
-.card-dark-style{
```
Comment (demianruban) [inline style.css:92]: this style is in use by Nikita's code ([contacts.html](https://github.com/demianruban/openvision-frontend/blob/main/contacts.html))

File: style.css [hunk 2]
```diff
@@ -33,69 +33,56 @@ hr{
...
-    margin: 1px;
-    border-radius: 1em;
-    padding: 1em;
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
+  border-radius: 5px;
 }
-
-.imagine-style{
```
Comment (demianruban) [inline style.css:99]: this style is also in use by Nikita's code ([contacts.html](https://github.com/demianruban/openvision-frontend/blob/main/contacts.html))

**Review by demianruban — CHANGES_REQUESTED** [review]
We use indentation with 4 spaces. Please edit that in style.css

**Comment by Tigee03** [comment]
Code formatted

**Pull request merged**